script/localstorage.js

```diff
@@ -0,0 +1,22 @@
+const contactForm = document.querySelector('#contact-form');
+const fullName = document.querySelector('#name');
+const emailAddress = document.querySelector('#email');
+const message = document.querySelector('#message');
+
+function dataStorage() {
+  const user = {
+    userFullName: fullName.value,
+    userEmail: emailAddress.value,
+    userMessage: message.value,
+  };
+
+  localStorage.setItem('userData', JSON.stringify(user));
+}
+
+contactForm.addEventListener('focusout', dataStorage);
+
+const userDataObject = JSON.parse(localStorage.getItem('userData'));
+
+fullName.value = userDataObject.userFullName;
+emailAddress.value = userDataObject.userEmail;
+message.value = userDataObject.userMessage;
```

Comment:
- [ ] Kindly I have tried to add some data but it didn't get saved in the local storage, It should be saved when the user add some data to the inputs even if he doesn't submit it, it should be saved too.

![image](https://user-images.githubusercontent.com/35535130/156770198-6ff2f3cf-b892-489e-9a41-110cf253ec16.png)